addons/osfstorage/models.py

```diff
@@ -342,6 +352,14 @@ def is_checked_out(self):
                 pass
         return False
 
+    @property
+    def is_preprint_primary(self):
```

Comment:
@alexschiller For now, just make a DjangoLE ticket to investigate optimizing this.